### DISCUSSION TITLE
bootstrap very nice

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#32504] [General] [answered]
I hope that the authors of bootstrap can quickly produce the RC version of bs5, because bootstrap is great

**Comment by florianlacreuse** [comment, marked as answer]
Dreams come true, there is a BS 5 final version since last week!

**Comment by hwang1588** [comment]
what is mean RC?

**Reply by florianlacreuse** [reply]
RC usually means release candidate.

**Reply by hwang1588** [reply]
@florianlacreuse

 Thank you!
